packages/adapter-static/README.md

```diff
@@ -38,6 +38,25 @@ The directory to write static assets (the contents of `static`, plus client-side
 
 Specify a fallback page for SPA mode, e.g. `index.html` or `200.html` or `404.html`.
 
+### outputFileName
+
+A optional function that is called per entry to return the destination to write to.
+
+Default value:
```

Comment:
Rather than include the code here, I would describe it textually like:

>By default, prerendered html pages are output as `index.html` files in the directory matching the URL path.